.azure-pipelines/azure-pipelines-linux.yml

```diff
@@ -14,6 +14,14 @@ jobs:
         CONFIG: linux_python2.7
         UPLOAD_PACKAGES: True
         DOCKER_IMAGE: condaforge/linux-anvil-comp7
+      linux_python3.6:
```

Comment:
Should we also add support for 3.5 testing? Beam supports it.

Reply:
I think conda-forge only maintains packages for the two latest Python versions (plus Python 2.7).